### PR TITLE
Fix FSAA dropdown option reset after changing another dropdown option

### PR DESCRIPTION
**Pull request by kilbith**
Also white spaces cleanup.

### CODE REVIEW

File: builtin/mainmenu/tab_settings.lua [hunk 1]
```diff
@@ -64,13 +64,13 @@ local dd_antialiasing_labels = {
 	fgettext("None"),
 	fgettext("2x"),
 	fgettext("4x"),
-	fgettext("8x"),
+	fgettext("8x")
```
Comment (est31) [inline builtin/mainmenu/tab_settings.lua]: Don't remove the comma, its bad style. If we want to add additional elements, we don't want to modify this line.

File: builtin/mainmenu/tab_settings.lua [hunk 2]
```diff
@@ -104,13 +104,12 @@ local function getAntialiasingSettingIndex()
...
 
 local function antialiasing_fname_to_name(fname)
-   for i=1, #(dd_antialiasing_labels) do
-      if fname == dd_antialiasing_labels[i] then
-         return antialiasing[2][i]
-      end
-   end
-
-   return "0"
+	for i=1, #(dd_antialiasing_labels) do
```
Comment (est31) [inline builtin/mainmenu/tab_settings.lua]: `i = 1` please

**Comment by est31** [comment]
see comments otherwise :+1:

**Comment by kilbith** [comment]
Comments fixed :)

**Comment by nerzhul** [comment]
:+1:

**Comment by est31** [comment]
ffd0ef16c41e85dc024760855bc54f116cfeebcf

**Pull request closed without merging**